mk_verificator/tests/system/single_vip/test_single_vip.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from mk_verificator import utils
+
+
+@pytest.mark.parametrize(
+    "group",
+    utils.get_groups(utils.get_configuration(__file__))
+)
+def test_single_vip(local_salt_client, group):
+    local_salt_client.cmd(group, 'saltutil.sync_all')
+    nodes_list = local_salt_client.cmd(group, 'grains.item', ['ipv4'])
+
+    ipv4_list = []
+
+    for node in nodes_list:
+        ipv4_list.extend(nodes_list.get(node).get('ipv4'))
+
+    ipv4_list.sort()
+    ipv4_list = list(filter(lambda ip: ip != '127.0.0.1', ipv4_list))
```

Comment:
let's keep this one and finalize, that's not critical for other stuff to merge it right now ;-)